android/src/main/java/ie/gov/tracing/common/Events.kt

```diff
@@ -59,10 +60,12 @@ class Events {
         fun raiseEvent(eventName: String, eventValue: ReadableMap?): Boolean {
             Log.d(TAG, eventName)
             if(!allowed(eventName)) return false
+            if(eventValue == null) return false
             val map = Arguments.createMap()
-            
+            val eventMap = WritableNativeMap()
+            eventMap.merge(map)
             try {
-                map.putMap(eventName, eventValue)
+                map.putMap(eventName, eventMap)
```

Comment:
The eventValue that was passed in is never added to the map

Reply:
:+1:  yes it was intended as `eventMap.merge(eventValue)`, this is typo, fixing that in your branch